pygameMenu/menu.py

```diff
@@ -260,18 +259,24 @@ def __init__(self,
         # Init mouse
         self._mouse = mouse_enabled
 
-    def add_option(self, element_name, element, *args):
+    def add_option(self, element_name, element, *args, align=''):
```

Comment:
This is a syntax error in python2, maybe indicate in readme that only Python3 is supported?

Reply:
@anxuae humm, by this only error I don't feel necessary to break compatibility. What else could i do? Maybe using **kwargs..

Reply:
kwargs works really good, by this weekend I'll work on README and minor stuff to publish v2.0